prime-angular-frontend/src/app/modules/enrolment/pages/organization/organization.component.html

```diff
@@ -24,7 +24,7 @@
         <div class="col col-sm-10 py-3"
              formArrayName="organizations">
 
-          <ng-container *ngFor="let organization of organizations.controls; let i = index"
+          <ng-container *ngFor="let organization of organizations.controls; let i = index;"
```

Comment:
Oh yay does this fix the weird code issue in VSCode?

Reply:
Yah, apparently how VSCode highlights markup now sees this as invalid JS in the template